dbt_subprojects/dex/models/_projects/zeroex/arbitrum/zeroex_arbitrum_settler_addresses.sql

```diff
@@ -2,9 +2,15 @@
     config(
         schema = 'zeroex',
```

Comment:
```suggestion
        schema = 'zeroex_arbitrum',
```

Reply:
just realized we had blockchain in alias instead of schema like normal, can we update all to this format?